prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/pages/IepHistory.groovy

```diff
@@ -6,7 +6,7 @@ class IepHistory  extends Page {
     static url = '/offenders/A1234AC/iep-level'
 
     static at = {
-        pageTitle == 'IEP History for NORMAN BATES'
+        pageTitle == 'IEP History for Norman Bates'
```

Comment:
@Nimphal can you confirm you are ok with this? 

This matches how we do things for scheduled events: 
<img width="594" alt="Screenshot 2019-05-15 at 09 31 06" src="https://user-images.githubusercontent.com/1517745/57760829-45ffca80-76f4-11e9-85de-7d8918e35134.png">

Reply:
Yeap, happy.

Reply:
I'll change the `history` to be lower case